fff

```diff
@@ -145,7 +145,11 @@ read_dir() {
     # If '$PWD' is '/', unset it to avoid '//'.
     [[ $PWD == / ]] && PWD=
 
+    max_len=1
     for item in "$PWD"/*; do
+        if [[ ${#item} -gt $max_len ]]; then
```

Comment:
What happens if `max_len` is larger than the window width? It might be simpler to just right align the size output so it hugs the right side of the terminal.

Reply:
Agree, fixed